src/components/Actions/GsuiteRoute.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useRecoilValue } from "recoil";
+import { auth_gsuite } from "../../api/state";
+import { Route, Redirect, withRouter } from "react-router-dom";
+
+const GsuiteProtectedRoute = ({ Component, ...rest }: any) => {
+  const auth_status = useRecoilValue(auth_gsuite);
+
+  return (
+    <Route
```

Comment:
When I previously wrote this component the interface was a little different from the standard Route itself. Could you make sure that that in this update that they're kept as close to the same as possible? Basically whether someone decides to use `<Route ...>` or `<GsuiteProtectedRoute ...>` the internal syntax should be as similar as possible.